src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -34,24 +36,55 @@
 public class Helios {
 
   public static HashMap<String, String> tokenCache = new HashMap<String, String>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "tokenCache" object should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make tokenCache a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "public static tokenCache" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)